### PR TITLE
`synapse` - `docs example improvement`

### PR DESCRIPTION
**Pull request by shu-ying789**

### CODE REVIEW

File: website/docs/r/synapse_sql_pool_vulnerability_assessment.html.markdown [hunk 1]
```diff
@@ -29,6 +29,11 @@ resource "azurerm_storage_account" "example" {
   is_hns_enabled           = "true"
 }
 
+resource "azurerm_storage_container" "example" {
+  name                 = "example"
+  storage_account_name = azurerm_storage_account.example.name
+}
```
Comment (stephybun) [inline website/docs/r/synapse_sql_pool_vulnerability_assessment.html.markdown:35]: I don't see this referenced anywhere else in the example config so why are we adding this here?

File: website/docs/r/synapse_workspace_vulnerability_assessment.html.markdown [hunk 2]
```diff
@@ -29,6 +29,11 @@ resource "azurerm_storage_account" "example" {
   is_hns_enabled           = "true"
 }
 
+resource "azurerm_storage_container" "example" {
+  name                 = "example"
+  storage_account_name = azurerm_storage_account.example.name
+}
+
```
Comment (stephybun) [inline website/docs/r/synapse_workspace_vulnerability_assessment.html.markdown:36]: Same here

**Review by stephybun — CHANGES_REQUESTED** [review]

**Comment by shu-ying789** [comment]
@stephybun I added this resource because it is referenced by storage_container_path.

**Review by stephybun — APPROVED** [review]
LGTM 🥟

**Pull request merged**